.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+# These owners will be the default owners for everything in the repo. Unless a later match takes precedence, @jph00 will 
+# be requested for review when someone opens a pull request.
+
+# Overall repository reviewers added to every PR
+* @LysandreJik @osanseviero
```

Comment:
My understanding from "These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence," is that since we have (near) full coverage, neither Lysandre nor me will be added to every PR. 

Also I don't think @LysandreJik wants to review everything in her, I think he should have same owners as you, since he mostly works in the `huggingface_hub` part of this repo